docs/pages/identity-governance/access-requests/automatic-reviews.mdx

```diff
@@ -90,43 +100,76 @@ does not support wildcard or regular expressions.
   by the IdP.
 </Admonition>
 
-## Step 3/4. Create automatic review rule
+## Role-based automatic reviews
 
-Next, define the automatic review rule via the Teleport Web UI.
+To demonstrate role-based automatic reviews, create the following Access Automation
+Rule via the Teleport Web UI:
 
 1. Go to **Identity Governance** -> **Access Requests** -> **Set Up Access Automation Rules**
 2. Click **Create New Access Automation Rule** -> **Automatic Review Rule**
 3. Configure the rule and set:
-   - **Name of roles to match**: `access`
-   - **User Traits**: `team: demo`
+   - **Name of requested roles to match**: `access`
+   - **User Traits to match**: `team: demo`
    - **Review decision**: `APPROVED`
 4. Click **Create Rule**
 
-![Create Automatic Review Rule Editor](../../../img/access-controls/automatic-reviews/create-automatic-review-rule.png)
-
-This automatic review rule will ensure that Access Requests for the `access`
-role will be automatically reviewed for approval if the Teleport user traits
-requirements are satisfied. In this case, any user with the traits `team: demo`
-will satisfy the requirement.
-
-## Step 4/4. Verify automatic review rule
+This rule automatically approves Access Requests for the `access` role if the
+user has the trait `team: demo`.
 
 To verify the new automatic review rule, create an Access Request via the Teleport
-Web UI.
+Web UI:
+
 1. Log in as `demo-access-requester`
-2. Go to **Access Requests** and click **New Access Request**
+2. Go to **Identity Governance** -> **Access Requests** and click **New Access Request**
 3. Change the request type from **Resources** to **Roles**
 4. Add the `access` role to the Access Request
 5. Click **Proceed to Request**, then **Submit Request**
 
-![Create Access Request](../../../img/access-requests/new-role-request.png)
-
 At this point, the new Access Request should have been created, automatically
 reviewed, and transitioned into an `APPROVED` state. Navigate **Back to Listings**
 and verify the Access Request status. It might take a second for the review to
 process, so you may have to refresh the page.
 
-![View Access Requests](../../../img/access-controls/automatic-reviews/view-access-requests.png)
+## Resource-based automatic reviews
+
+Before creating an Access Automation Rule, ensure that you have at least one
+labeled SSH server connected to your Teleport cluster:
+
+```code
+# List your SSH servers to identify available labels:
+$ tsh ls
+Node Name   Address    Labels
+----------- ---------- -------------------------------
+teleport-00 ⟵ Tunnel  env=demo
+```
+
+To demonstrate resource-based automatic reviews, create the following Access Automation
```

Comment:
Same as above, instead of "resource-based automatic reviews", reiterate the use-case that is being demonstrated.